rsts/index.rst

```diff
@@ -54,27 +54,208 @@ Created at `Lyft <https://www.lyft.com/>`__ in collaboration with Spotify, Freen
 
 The core unit of execution in Flyte is the ``task``, which you can easily write with the Flytekit Python SDK:
 
-.. code:: python
-   
-   @task
-   def greet(name: str) -> str:
-       return f"Welcome, {name}!"
+.. tabs::
 
-You can compose one or more tasks to create a ``workflow``:
+    .. tab:: python
 
-.. code:: python
+        .. code:: python
 
-   @task
-   def add_question(greeting: str) -> str:
-       return f"{greeting} How are you?"
-   
-   @workflow
-   def welcome(name: str) -> str:
-       greeting = greet(name=name)
-       return add_question(greeting=greeting)
+           @task
```

Comment:
While we are at it, can we also add comments to python 😬 ?

Reply:
@EngHabu I am actually thinking of removing comments from Java. To make it more concise, but just use good code. But the actual example should have comments - and we should link to the actual example

Reply:
Can we simplify the comments to explain "what and why things need to be written the way it's for Flyte" as opposed to adding "java docs" everywhere?

Reply:
@EngHabu Most javadocs are not related to Flyte concepts, we can remove all javadocs and keep the comments in the workflow file.

Reply:
excellent!